Lib/test/test_buffer.py

```diff
@@ -3183,6 +3184,7 @@ def cmptest(testcase, a, b, m, singleitem):
                             self.assertEqual(m.tobytes(), a.tobytes())
                             cmptest(self, a, b, m, singleitem)
 
+    @warnings_helper.ignore_warnings(category=DeprecationWarning)  # gh-80480 array('u')
     def test_memoryview_compare_special_cases(self):
```

Comment:
Would it be possible to move the "u" test in a new method?

Reply:
Yep, now moved to its own method. Makes it easier to remove when the time comes.